packages/accounts-base/accounts_common.js

```diff
@@ -204,7 +204,7 @@ export class AccountsCommon {
     const VALID_KEYS = ["sendVerificationEmail", "forbidClientAccountCreation", "passwordEnrollTokenExpiration",
                       "passwordEnrollTokenExpirationInDays", "restrictCreationByEmailDomain", "loginExpirationInDays",
                       "loginExpiration", "passwordResetTokenExpirationInDays", "passwordResetTokenExpiration",
-                      "ambiguousErrorMessages", "bcryptRounds", "defaultFieldSelector"];
+                      "ambiguousErrorMessages", "bcryptRounds", "defaultFieldSelector", "useLowerCaseEmails"];
```

Comment:
Please add description about this new option above in the comment so that it shows up in documentation.

Reply:
Hi I have made the necessary changes and create a new commit